.github/workflows/build_and_push_docker_latest.yml

```diff
@@ -0,0 +1,34 @@
+name: build_test_and_push_docker
+
+on:
+  push:
+    branches:
+      # Only build containers when pushing to main
+      - "main"
+
+env:
+  LATEST_TAG: dtcenter/ccpp-scm:latest
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
```

Comment:
Maybe change to "ubuntu-22.04", the latest OS can change and cause hiccups.

Reply:
Good point, updated